src/handler/commands.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+// Core Modules
+let fs = require("fs");
+let path = require("path");
+let log = require("../utils/logger");
+
+function loadCommands(srcDir) {
+    const commandRoot = path.resolve(srcDir);
+
+    const res = new Map();
+
+    for(const commandFile of fs.readdirSync(commandRoot)) {
+        const fullCommandFile = path.join(commandRoot, commandFile);
+
+        if (fs.statSync(fullCommandFile).isDirectory() || path.extname(fullCommandFile).toLowerCase() !== ".js") {
+            continue;
+        }
+
+        log.info(`Loading "${fullCommandFile}"`);
+        res.set(commandFile.toLowerCase(), require(fullCommandFile));
```

Comment:
Siehe oben. Oder hier halt 
```suggestion
        res.set(commandFile.toLowerCase().replace(/\.js$/, ""), require(fullCommandFile));
```
oder
```suggestion
        res.set(path.parse(commandFile).name, require(fullCommandFile));
```

Reply:
1 nicer fund